src/components/orderdetail/OrderDetails.js

```diff
@@ -0,0 +1,71 @@
+import "./OrderDetail.css";
+import { useSelector, useDispatch } from "react-redux";
+import {
+  addIngredient,
+  removeIngredient,
+} from "../../redux/slices/burgerBuilderSlice";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import CheckoutModal from "../dialog/Dialog";
+function OrderDetails() {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+
+  const openModal = () => {
+    setIsModalOpen(true);
+  };
+
+  const closeModal = () => {
+    setIsModalOpen(false);
+  };
+  const isLoggedIn = useSelector((state) => state.authSlice.isLoggedIn);
+  const burgerIngredients = ["salad", "bacon", "cheese", "meat"];
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const { totalPrice, addedIngredients } = useSelector(
+    (state) => state.BurgerBuilderSlice
+  );
+
+  function handleAddIngridiant(Ingredients) {
```

Comment:
function params starts with small letters, why is I captial here and in other functions.